api/src/main/java/org/pmiops/workbench/sam/SamConfig.java

```diff
@@ -0,0 +1,32 @@
+package org.pmiops.workbench.sam;
+
+import org.broadinstitute.dsde.workbench.client.sam.ApiClient;
+import org.broadinstitute.dsde.workbench.client.sam.api.TermsOfServiceApi;
+import org.pmiops.workbench.auth.UserAuthentication;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.web.context.annotation.RequestScope;
+
+@org.springframework.context.annotation.Configuration
+public class SamConfig {
```

Comment:
new file.  My intention is to be fully standardized here, nothing custom.  I copied it from other similar configs.